docs(models): clarify byok and subscriptions (#638)
tembo/docs: features/max.mdx

Available on all plans, including the free tier. [Read more about Tembo Max](https://tembo.io/max).

> BYOK is also available on all plans from **Settings -> Models -> API Keys**. Paid Tembo plans can connect supported agent subscriptions, including Codex, from **Settings -> Connected accounts -> LLM subscriptions**.
> 

Author: ```suggestion
BYOK is also available on all plans from **Settings -> Models -> API Keys**. All Tembo users can connect supported agent subscriptions, including Codex, from **Settings -> Connected accounts -> LLM subscriptions**.

```

| Kimi K2           | `bedrock-kimi-k2-thinking` | `proxy.tembo.io/anthropic`    |
| Grok              | `grok`                     | Cursor API                    |

> Tembo-managed Anthropic and Google models require no additional provider keys. Some OpenAI models require BYOK with your own OpenAI API key, and paid Tembo plans can use supported agent subscriptions such as Codex where available.

Author: ```suggestion
Tembo-managed Anthropic and Google models require no additional provider keys. Some OpenAI models require BYOK with your own OpenAI API key or Codex subscription.
```
